Documentation/Authenticators/Authentication-Modules.md

```diff
@@ -86,7 +86,7 @@ GRANT SHOW DATABASES ON *.* TO 'maxscale'@'maxscalehost';
 
 ## Limitations and troubleshooting
 
-When a client logs in to MaxScale, MaxScale sees the client's IP address. When
+When a client logs into MaxScale, MaxScale sees the client's IP address. When
```

Comment:
Changes rom `in to` to `into` have been fairly controversial. I'm happy to drop this or anything else.